### PR TITLE
[Sprint: 42] XD-2690 - Filter lib files other than .zip and .jar

### PR DESCRIPTION
**Pull request by dturanski**

### CODE REVIEW

File: spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java [hunk 1]
```diff
@@ -37,7 +37,11 @@
  */
 public class ModuleUtils {
 
-	private static final AsciiBytes LIB = new AsciiBytes("lib/");
+	private static final String LIB = "lib/";
+
+	public static final String DOT_JAR = ".jar";
+
+	public static final String DOT_ZIP = ".zip";
```
Comment (ericbottard) [inline spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java]: why even accept .zip? We won't add them to the CP currently

Comment (mbogoevici) [inline spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java]: `.zip` is a legitimate extension for a CP element. See http://docs.oracle.com/javase/7/docs/technotes/tools/windows/classpath.html. I don't see what we could possibly gain by not accepting them. 

@ericbottard can you clarify the second part of the comment: org.springframework.xd.module.support.ModuleUtils#createModuleClassLoader(org.springframework.core.io.Resource, java.lang.ClassLoader, boolean) doesn't seem to do any filtering of archives of its own.

Comment (ericbottard) [inline spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java]: My bad, I was under the impression we only looked up .jar
So yeah, we shall accept .zip
The more, the merrier...

Comment (mbogoevici) [inline spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java]: OK :) I'll LGTM it, and add it to the merge pipeline.

**Comment by mbogoevici** [comment]
Merged w/https://github.com/spring-projects/spring-xd/commit/97aa52a5a610572fa8179d9f6a4790122bc3a701

**Pull request closed without merging**